testsuite/cluster-test/src/cluster_swarm/cluster_swarm_kube.rs

```diff
@@ -57,6 +60,11 @@ impl ClusterSwarmKube {
         image_tag: &str,
         delete_data: bool,
     ) -> Result<Vec<u8>> {
+        let cfg_fullnode_seed = if num_fullnodes > 0 {
```

Comment:
What will happen if you just put is unconditionally as `let cfg_fullnode_seed= CFG_FULLNODE_SEED`? Why it has to be "" if there is no full nodes?

Reply:
Because `docker-run-dynamic.sh` expects a fullnode network when `CFG_FULLNODE_SEED` is set. Setting `CFG_NUM_FULLNODES` to 0 gives an error.